src/steamship/agents/examples/example_chatbot_with_replicate.py

```diff
@@ -0,0 +1,36 @@
+from steamship.agents.basic_chat import BasicChatAgent
+from steamship.agents.llms.replicate import ChatReplicate
+from steamship.agents.schema.message_selectors import MessageWindowMessageSelector
+from steamship.agents.service.agent_service import AgentService
+from steamship.utils.repl import AgentREPL
+
+
+class MyAssistant(AgentService):
```

Comment:
I don't want to derail this PR but would this fit better in the MMAS?

I think there was a recent PR where there was discussion about where examples should live in general but I'd have to dig it up.

Reply:
We, at the very least, need an example `AgentService` that we are testing in this codebase.

Which raises the question of "where are the tests" for this code?

Reply:
Now that the MMAS has a pretty different structure, there are a few moves we could choose to make if we wanted:

- Relocate the agents for testing into the `tests/assets` folder, which is where some other things like those are stored.
- Keep them here, but remove the ones that weren't used for testing.

Seems like the second one is probably worthwhile in a new PR. The first one I'm unsure if it's a win